docs/admin-manual/stream-configuration.md

```diff
@@ -17,6 +17,12 @@ Setting a higher bitrate for your output stream will only benefit your listeners
 
 :::
 
+:::caution
+
+The version of the liquidsoap playout handler distributed in supported operating systems doesn't support AAC streaming output. If you want to stream AAC, you will need to replace the liquidsoap package with a version that supports AAC. See this [tutorial](./tutorials/setup-liquidsoap-aac-streaming.md) for more information.
```

Comment:
Be specific here - the issue is Liquidsoap version 1.4.3 shipped in Bullseye and 1.4.1 shipped in Focal. Other distros versions are not affected